ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/dot1x.j2

```diff
@@ -20,6 +20,29 @@
 {%         set protocol_lldp_bypass = dot1x.protocol_lldp_bypass | arista.avd.default('-') %}
 {%         set dynamic_authorization = dot1x.dynamic_authorization | arista.avd.default('-') %}
 | {{ system_auth_control }} | {{ protocol_lldp_bypass }} | {{ dynamic_authorization }} |
+{%         if dot1x.mac_based_authentication or dot1x.radius_av_pair is arista.avd.defined %}
```

Comment:
Do you really need the outer if? It looks like line 24 (for mac_based_authentication) and 34 (for radius_av_pair) if are sufficient

Reply:
Fixed